draft-barnes-acme.md

```diff
@@ -1449,28 +1477,30 @@ problem document {{I-D.ietf-appsawg-http-problem}}.
 
 All additional fields are specified by the Challenge type.  The server MUST
 ignore any values provided in the "uri", "status", "validated", and "error"
-fields of a Response payload.  Additionally, if the server sets a Challenge's
-"status" to "invalid", it SHOULD also include the "error" field to help the
-client diagnose why they failed the challenge.
+fields of a Response payload.  If the server sets a Challenge's "status" to
+"invalid", it SHOULD also include the "error" field to help the client diagnose
+why they failed the challenge.
 
 Different challenges allow the server to obtain proof of different aspects of
 control over an identifier.  In some challenges, like Simple HTTP and DVSNI, the
-client directly proves control of an identifier.  In other challenges, such as
-Proof of Possession, the client proves historical control of the identifier, by
-reference to a prior authorization transaction or certificate.
+client directly proves its ability to do certain things related to the
+identifier.  In the Proof of Possession challenge, the client proves historical
+control of the identifier, by reference to a prior authorization transaction or
+certificate.
 
 The choice of which Challenges to offer to a client under which circumstances is
-a matter of server policy.  A server may choose different sets of challenges
+a matter of server policy.  A CA may choose different sets of challenges
 depending on whether it has interacted with a domain before, and how.  For
 example:
 
-| Domain status                                 | Challenges typically sufficient for (re)Authorization |
-|:----------------------------------------------|:------------------------------------------------------|
-| No known prior certificates or ACME usage     | Domain Validation (DVSNI or Simple HTTP)              |
-| Existing valid certs, first use of ACME       | DV + Proof of Possession of previous CA-signed key    |
-| Ongoing ACME usage                            | PoP of previous Authorized key                        |
-| Ongoing ACME usage, lost Authorized key       | DV + PoP of ACME-certified Subject key                |
-| ACME usage, all keys and recovery tokens lost | Proof of legal identity of the site owner             |
+* New domain with no known certificates: Domain Validation (DVSNI or Simple
```

Comment:
Nit: Could use the Kramdown definition list construction like you did up at line 191-198.